packages/instance-model/lib/model.js

```diff
@@ -365,7 +365,6 @@ const InstanceModel = AmpersandModel.extend(
 
     removeAllListeners() {
       removeListenersRec(this);
-      VisitedModels.deleteAll();
```

Comment:
This only worked when `core-js` was loaded as a polyfill, because this method didn’t make it into the standardized variant of `WeakSet` :slightly_smiling_face:

Reply:
Oh wow, TIL